interactive_textinput_printer.go

```
@@ -200,13 +215,19 @@ func (p InteractiveTextInputPrinter) updateArea(area *AreaPrinter) string {
 		p.cursorYPos = 0
 	}
 	areaText := p.text
+
 	for i, s := range p.input {
 		if i < len(p.input)-1 {
 			areaText += s + "\n"
 		} else {
 			areaText += s
 		}
 	}
+
+	if p.hide {
```

```suggestion
	if p.hide || p.mask != "" {
```

---

I think this setting makes more sense.

Then you can call it like this:

```
p := pterm.DefaultInteractiveTextInput.WithMask("*")
```

instead of
```
p := pterm.DefaultInteractiveTextInput.WithMask("*").WithHide(true)
```

wdyt?

---

we would need to set the default value of `mask` to `""` tho..

---

I updated the PR with the changes

---

Perfect!